src/Notepads/Notepads.csproj

```diff
@@ -225,6 +225,7 @@
     <Compile Include="Models\TextFile.cs" />
     <Compile Include="Extensions\StringExtensions.cs" />
     <Compile Include="Utilities\ThreadUtility.cs" />
+    <Compile Include="XboxGameBarForegroundWorkerWrapper.cs" />
```

Comment:
I wasn't sure exactly where you would want this file, it didn't seem to fit cleanly in any of your existing folders. Let me know what you think.

Reply:
You can create a Widget folder and put xbox code/class there. Maybe also create a IForegroundWorkerWrapper interface and have a default implementation for handling normal (non-wdiget) scenario (ForegroundWorkerWrapper.cs). 

Based on App.IsGameBarWdiget and _widget != null, you can then create a XboxGameBarForegroundWorkerWrapper class on the fly to handle widget file opening/saving or alternatively, use ForegroundWorkerWrapper for normal path. Otherwise even for normal Notepads instance, game bar DLLs will be loaded when opening file save/open dialogs.